test/Cargo.toml

```diff
@@ -7,7 +7,7 @@ publish = false
 
 [dependencies]
 anyhow = "1.0.41"
-forc = { version = "0.11.0", path = "../forc", features = ["test"], default-features = false }
+forc = { version = "0.12.0", path = "../forc", features = ["test"], default-features = false }
```

Comment:
This test crate isn't published, so I don't think it needs a `version` here, it can just use a local path. Can you file an issue to remove the unnecessary `version`?

Reply:
Good point. Done.